setup.cfg

```diff
@@ -10,6 +10,8 @@ license = Apache-2.0
 zip_safe = False
 include_package_data = True
 packages = find:
+package_dir =
+    =src
```

Comment:
Is this correct syntax?

Reply:
Yes, it is. See https://setuptools.pypa.io/en/latest/userguide/package_discovery.html